src/OSPSuite.Core/Serialization/Xml/ParameterValuesXmlSerializer.cs

```diff
@@ -40,7 +40,9 @@ protected override XElement TypedSerialize(ParameterValues parameterValues, Seri
             return element;
 
          var doubleSerializer = SerializerRepository.SerializerFor(parameterValues.Percentiles);
-         element.Add(doubleSerializer.Serialize(parameterValues.Percentiles, serializationContext));
+         var percentilesElement = doubleSerializer.Serialize(parameterValues.Percentiles, serializationContext);
+         percentilesElement.Name = Constants.Serialization.PERCENTILES;
```

Comment:
This node did not have the right name => E.g. it was not deserialized properly

Reply:
@msevestre why is percentile serialized at all? Isn't it always calculated from the distribution and value of the parameter?